### PR TITLE
Feature - Merge of topics for cleanup

### PR DESCRIPTION
**Pull request by clmedders**
## Summary

- This PR will handle the section of merging topics. 
- Old URLs were added to Wayback Machine 
- Total topics are now at 440

Spreadsheet for topics that are being merged is listed in the ticket named `DG - Topics: Remove ROT topics`

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/feature/cm-merge-topics/)

### Solution
Removed old slug and replaced any ones listed with the "new-merged" slug, and renamed ones as needed.

### How To Test

1. Review spreadsheet and verify that URLs are no longer available 
2. Check that all listed URLs are captured within this PR 

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

Tagging Supervisor - @rayestrada

### CODE REVIEW

**Review by mejiaj — APPROVED** [review]
Builds passing, no issues found on `/topics`. Confirmed tags were removed.

**Review by nick-mon1 — DISMISSED** [review]
Looks good, thanks!

- [x] build runs
- [x] checked each topic
- [x] no duplicates

**Comment by nick-mon1** [comment]
@clmedders We need to update `{{< ref "/customer-experience.md" >}}` to `{{< ref "/communities/customer-experience.md" >}}`

File: content/news/2015/12/2015-12-28-ocsits-2015-customer-survey-what-we-learned.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ topics:
   - customer-service
   - dap
   - digital-analytics-program
-  - ecpic
+  - budgeting-and-performance
```
Comment (clmedders) [inline content/news/2015/12/2015-12-28-ocsits-2015-customer-survey-what-we-learned.md:15]: Budgeting and performance is the one that ecpic was merged into. If you take a look at line 418 in the spreadsheet it shows that the merge of the 2 topics

**Pull request merged**